pandas/core/sorting.py

```diff
@@ -92,8 +95,13 @@ def get_indexer_indexer(
         ):
             return None
 
+        # error: Argument "ascending" to "nargsort" has incompatible type
+        # "Union[Sequence[Union[bool, int]], bool, int]"; expected "bool"
         indexer = nargsort(
```

Comment:
At this point, we know that using a `list` would be inappropriate, so we also know that the type of `ascending` is `bool`, and I think an `assert(isinstance(ascending, bool))` would then avoid the ignore.  Or you could use `cast` and add a comment.

Reply:
Used a cast+comment in the next commit.